readers/common/src/main/java/com/criteo/hadoop/garmadon/protobuf/ProtoConcatenator.java

```diff
@@ -91,7 +91,13 @@ protected ProtoConcatenator() {
                 return concatMap;
             },
             (entry, eventMap) -> {
-                eventMap.put(entry.getKey().getName(), getRealFieldValue(entry.getValue()));
+                Object value;
+                if ("DOUBLE".equals(entry.getKey().getType().name())) {
+                    value = Double.NEGATIVE_INFINITY == (Double) entry.getValue() ? -Double.MAX_VALUE : getRealFieldValue(entry.getValue());
```

Comment:
Use `Double.MIN_VALUE` instead

Reply:
For NEGATIVE_INFINITY is more closer to -Double.MAX_VALUE than Double.MIN_VALUE

Reply:
-Double.MAX_VALUE is the correct value (as Double.MIN_VALUE is the smallest positive nonzero value of type double.